relayer/src/chain/cosmos/events.rs

```diff
@@ -0,0 +1,57 @@
+//! Extension events specific to chains followed via Cosmos SDK.
```

Comment:
I'm not quite sure what "followed via Cosmos SDK" means. Would "chains that adhere to the Cosmos SDK" be more accurate?

Reply:
_chains built on the Cosmos SDK_ ?